### PR TITLE
MP-8785: Added the following fields to AllNoTypes entity:

### PR DESCRIPTION
**Pull request by originalethanhunt**
AreaField
DayOfWeekField
DistanceField
EmailField
EncryptedTextField
GuidField

### CODE REVIEW

**Review by dejobratic — APPROVED** [review]

File: src/Nox.Types.Abstractions/Enums/NoxType.cs [hunk 1]
```diff
@@ -128,10 +128,10 @@ public enum NoxType : uint
     [SimpleType(typeof(string))]
     DateTimeSchedule = 3271762546,
 
-    [SimpleType(typeof(byte))]
+    [SimpleType(typeof(int))]
```
Comment (rochar) [inline src/Nox.Types.Abstractions/Enums/NoxType.cs]: To be aligned with Value object
```suggestion
    [SimpleType(typeof(ushort))]
```

**Review by rochar — CHANGES_REQUESTED** [review]
Looks good, just a small comment

File: src/Nox.Types.Abstractions/Enums/NoxType.cs (hunk 1 above)
Comment (rochar) [reply src/Nox.Types.Abstractions/Enums/NoxType.cs]: update from main and regenerate the migrations

Comment (originalethanhunt) [reply src/Nox.Types.Abstractions/Enums/NoxType.cs]: done

**Review by rochar — APPROVED** [review]

**Pull request merged**